src/test/java/app/sami/languageWeb/language/LanguageRequestMapper.java

```diff
@@ -0,0 +1,18 @@
+package app.sami.languageWeb.language;
+
+import app.sami.languageWeb.language.dtos.LanguageGradeRequest;
+import app.sami.languageWeb.language.models.LanguageGrades;
+
+public class LanguageRequestMapper {
+
+    public static LanguageGradeRequest toLanguageGradeRequest(LanguageGrades languageGrades){
```

Comment:
you should simply put that into `languages/mapper` and use it in the actual code